### PR TITLE
[14.0][ADD] sale_order_line_chained_move_purchase

### PR DESCRIPTION
**Pull request by rousseldenis**
Adds a link to sale order line on purchase order line level through chained moves.

Depends on:

- [x] #1572 

Remove:

- [x] test-requirements.txt

### CODE REVIEW

**Comment by rousseldenis** [comment]
@xavier-bouquiaux

File: sale_order_line_chained_move_purchase/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2020 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```
Comment (xavier-bouquiaux) [inline sale_order_line_chained_move_purchase/__manifest__.py:2]: 2023

File: sale_order_line_chained_move_purchase/hooks.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Copyright 2021 ACSONE SA/NV
```
Comment (xavier-bouquiaux) [inline sale_order_line_chained_move_purchase/hooks.py:1]: 2023

File: sale_order_line_chained_move_purchase/tests/test_purchase_related_sale.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
+# Copyright 2020 ACSONE SA/NV
```
Comment (xavier-bouquiaux) [inline sale_order_line_chained_move_purchase/tests/test_purchase_related_sale.py:1]: 2023

File: sale_order_line_chained_move_purchase/models/purchase_order_line.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
+# Copyright 2021 ACSONE SA/NV
```
Comment (xavier-bouquiaux) [inline sale_order_line_chained_move_purchase/models/purchase_order_line.py:1]: 2023

File: sale_order_line_chained_move_purchase/models/sale_order_line.py [hunk 5]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2021 ACSONE SA/NV
```
Comment (xavier-bouquiaux) [inline sale_order_line_chained_move_purchase/models/sale_order_line.py:1]: 2023

File: sale_order_line_chained_move_purchase/tests/test_purchase_related_sale.py [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import SavepointCase
+
+
+class TestSaleOrderLineUpdateRoute(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.move_obj = cls.env["stock.move"]
```
Comment (xavier-bouquiaux) [inline sale_order_line_chained_move_purchase/tests/test_purchase_related_sale.py:11]: add : 
cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))

**Review by xavier-bouquiaux — COMMENTED** [review]
Good job @rousseldenis 

Just some minor changes

**Pull request closed without merging**